src/training/ea/lipizzaner_gan_trainer.py

```diff
@@ -139,30 +152,33 @@ def train(self, n_iterations, stop_event=None):
             self.batch_number = 0
             data_iterator = iter(loaded)
             while self.batch_number < len(loaded):
-            # for i, (input_data, labels) in enumerate(loaded):
+                # for i, (input_data, labels) in enumerate(loaded):
```

Comment:
Remove commented outline if it is not used

Reply:
Sure!